xknx/remote_value/remote_value.py

```diff
@@ -165,21 +166,18 @@ async def process(self, telegram: Telegram, always_callback: bool = False) -> bo
                 device_name=self.device_name,
                 feature_name=self.feature_name,
             )
+        decoded_payload = self.from_knx(_new_payload)
         self.xknx.state_updater.update_received(self)
-        if self.payload is None or always_callback or self.payload != _new_payload:
-            self.payload = _new_payload
+        if self.value is None or always_callback or self.value != decoded_payload:
+            self.value = decoded_payload
+            self.telegram = telegram
             if self.after_update_cb is not None:
                 await self.after_update_cb()
         return True
 
-    @property
-    def value(self) -> ValueType | None:
-        """Return current value."""
-        if self.payload is None:
-            return None
-        return self.from_knx(self.payload)
-
-    async def _send(self, payload: DPTPayloadType, response: bool = False) -> None:
+    async def _send(
+        self, payload: Union[DPTBinary, DPTArray, None], response: bool = False
```

Comment:
```suggestion
        self, payload: DPTPayloadType, response: bool = False
```
`payload` can't be optional here. We can't send a GroupValueWrite/Response without payload.
`_send()` is only called by `set()` - which has not changed so this should be fine the way it was?!